### DISCUSSION TITLE
Proofreading Tools

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#60] [Ideas]
In terms of open-source alternatives to Grammarly, [LanguageTool](https://github.com/languagetool-org/languagetool) might be viable (are there others?). Could it be incorporated into Tero Subtitler?

**Comment by GitGianluc** [comment]
I love LanguageTool, especially because it's one of the few tools (or maybe the only one I know) that work quite well with languages other than English!

**Reply by chenlung** [reply]
> I love LanguageTool, especially because it's one of the few tools (or maybe the only one I know) that work quite well with languages other than English!

I haven't used it myself, but it would be interesting to do. At the moment, I use Grammarly to check continuous scripts (i.e. without time codes or line breaks) so that oversights are caught better (e.g. a full in one entry followed by a capital letter in the next).
